graph/metafeeds_test.go

```diff
@@ -83,6 +85,134 @@ var metafeedsScenarios = []PeopleTestCase{
 			PeopleAssertHops("alice", 0, "alice-legacy", "some"),
 		},
 	},
+
+		// name: "main announces fake metafeed, should not work",
+		// name: "followed peer has newly announced metafeed",
+	{
+		name: "follow peer with metafeed announcement",
+		ops: []PeopleOp{
+			PeopleOpNewPeer{"alice-main"},
+			PeopleOpNewPeer{"bob-main"},
+			PeopleOpFollow{"alice-main", "bob-main"},
+			PeopleOpNewPeerWithAlgo{"bob-mf", refs.RefAlgoFeedBendyButt},
+			PeopleOpAnnounceMetafeed{"bob-main", "bob-mf"},
+			// question: do we need to add a new op for adding an existing subfeed?
```

Comment:
nope, this is fine.

there should be a seperate testcase for AddExisting on the metafeed.